### DISCUSSION TITLE
How to convert markdown to editor node?

### DISCUSSION THREAD

**Discussion by lorcan-codes** [mdx-js/mdx#2418] [Q&A]
Do you have any APIs to convert markdown to editor underlying nodes? One use case is to paste raw markdown on editor from https://markdown-it.github.io for example and it will convert this markdown to editor's node so that it displays them nicely.

**Comment by ChristianMurphy** [comment]
Welcome @lorcan-codes! 👋 
This is an XY Problem, and as such is difficult to answer effectively.
Please see https://mdxjs.com/community/support/ for recommendations on how help others help you.

Making an educated guess at what you want.
It sounds like you want a what you see is what you get editor experience (WYSIWYG) for MDX?
There are a few already:
* https://tina.io/docs/editing/markdown/
* https://www.mdx-blocks.com/
* https://areknawo.medium.com/wysiwyg-for-mdx-introducing-vrites-hybrid-editor-26f2fe7372e8
* https://mdxeditor.dev/
* https://github.com/jxnblk/ok-mdx
